.github/workflows/post_merge_checks.yml

```diff
@@ -23,6 +23,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r cicd_requirements.txt
+          pip install peekingduck --no-dependencies
```

Comment:
I checked with LP, I think it would be for this to test the in-development code so we can hopefully reduce the need to patch published packages.

I think it should be possible to script the local install with 
```
bash ./scripts/update_version.sh $VERSION
pip install . --no-dependencies
```
Probably can hard code `$VERSION` to something like `0.0.0.dev1` and check for that in the tests.

Example usage of `update_version.sh` can be found [here](https://github.com/aimakerspace/PeekingDuck/blob/dev/.github/workflows/publish_pypi.yml#L21)

Reply:
Updated post merge script to perform local install with a test version number that is read from `test_api.py` (the testing code here will also do a check for this version).

Reply:
Todo : duplicate the local install and testing into the daily drift test script.

Reply:
- Added api testing into daily drift test script.
- Had to separate test into Windows and Linux branches because `grep` is unavailable in Windows environment. In Windows branch, had to hardcode the Peeking Duck version number.
- Also added pip dependency caching to pre- and post-merge checks to close #537.

Before pip dependency caching:
![image](https://user-images.githubusercontent.com/13075447/143470717-112b70f3-edec-49ef-a651-ff0867bc5fbe.png)

After pip dependency caching:
![image](https://user-images.githubusercontent.com/13075447/143470894-4d7d7bf4-cbb2-4db3-8eb7-ad7de91ff0ec.png)